src/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+const ADD = 'ADD_BOOK';
+const REMOVE = 'REMOVE_BOOK';
```

Comment:
- [ ] Good job so far 👍 Kindly use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) to write your actions.
- [ ] Also I am not able to add or remove a book. Kindly implement this functionality too 💯

Reply:
👍 noted and fixed